.github/workflows/deployment-images.yaml

```diff
@@ -5,6 +5,18 @@ on:
       build-information-path:
         required: true
         type: string
+      docker-build-dir:
+        required: false
+        type: string
+        default: '.'
+      docker-build-file:
+        required: false
+        type: string
+        default: 'Dockerfile'
+      repository:
```

Comment:
Can you expand on why the repository name needs to be passed as a parameter? At first glance, the motivation for this is unclear.

Reply:
Indeed, "repository" is not correct name. We must choose any and use it consistently. How about "artifact"?

Reply:
I added more details on motivation in the issue section -> https://github.com/rentpath/actions-services-workflows/issues/6